docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It allows to view and edit BPMN 2.0 diagrams in the browser without any backend requirement.
+
+Feel free to play with it [here](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+There are two ways to include bpmn-js in your application:
+
+* An [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/) allowing you to quickly add BPMN to any web application.
+* The [npm](http://npmjs.com/package/bpmn-js) version, which is more complicated to set up but gives you access to individual library components and allows for easier extensibility.
+
+## Including the Root Element
+
+Both as a viewer or modeler, bpmn-js requires a root DOM element (a simple DIV) on which the rendering is performed. Simply, create a DIV element in the HTML file:
+
+```html
+<!-- BPMN diagram container -->
+<div id="canvas"></div>
+```
+
+## Including the Stylesheet
+
+We should include [diagram-js](/general/getting-started/diagram-js) CSS and [BPMN Icon Font](https://github.com/bpmn-io/bpmn-font) in order to ensure that everything looks good. There're several ways we can achieve that:
+
+We can include the CSS via `link` tag in our HTML file. Note that these CSS files are shipped with with the bpmn-js distribution under [dist/assets folder](https://unpkg.com/browse/bpmn-js/dist/assets/).
+
+```html
+<link rel="stylesheet" href="bpmn-js/dist/assets/diagram-js.css" />
+<link rel="stylesheet" href="bpmn-js/dist/assets/bpmn-font/css/bpmn.css" />
+```
+
+If we've install the bpmn-js via [NPM](#including-via-npm), we can additionally include these CSS files using `import` statement in our Javascript file:
+
+```javascript
+import "bpmn-js/dist/assets/diagram-js.css";
+import "bpmn-js/dist/assets/bpmn-font/css/bpmn-embedded.css";
+```
+
+## Using bpmn-js as Viewer
```

Comment:
I think I would like to have a short intro section and the following then as sub-sections. The section would be called `Alternatives to use bpmn-js`
```
The library is built in a way that it can be both a viewer and web modeler. Use the viewer to embed BPMN 2.0 into your applications and enrich it with your data. Use the modeler to create BPMN 2.0 diagrams inside your application.
```

(potentially add navigated viewer)

Reply:
I added the subsection however used a more engaging format (bulleted list) for the parent section. See the latest commit and let me know if that works for you as well.

Reply:
As Philipp mentionned, `Alternatives to use bpmn-js` could sound like we're suggesting different libraries to use, so I changed the title to `bpmn-js Use Cases`. I'm open to better ideas though.